lib/chore_runner/downloads.ex

```diff
@@ -0,0 +1,26 @@
+defmodule ChoreRunner.Downloads do
+  alias ChoreRunner.Downloads.StorageService
+
+  def create_download(filename, opts), do: StorageService.save_file(filename, opts)
+
+  def list_downloads do
+    with {:ok, local_downloads} <- StorageService.list_files(),
+         {results, _} <-
+           :rpc.multicall(Node.list(), ChoreRunner.Downloads.StorageService, :list_files, []) do
+      results
+      |> Enum.flat_map(fn
+        {:ok, files} -> files
+        _ -> []
+      end)
+      |> Kernel.++(local_downloads)
+    end
+  end
+
+  def delete_download(file) do
+    with {:error, _} <- StorageService.delete_file(file),
```

Comment:
I think the same could be done here as well https://github.com/pepsico-ecommerce/chore_runner/pull/10/files#r1115158834

Reply:
See above